src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-macos`
Can load a native assembly.Can load native dll
```powershell
Expected $null or empty, but got /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpoXbnDw.tmp/testDllNativeFolder/osx-x64;/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpoXbnDw.tmp/testDllNativeFolder/managed.dll.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 64
64:         (dir $root)  -join ";" | Should -BeNullOrEmpty
```
Can load a native assembly.Can load native dll
```powershell
Unable to find type [TestNativeClass].
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 76
76:         [TestNativeClass]::LoadNative() | Should -Not -BeNullOrEmpty

```